chain/network/src/tests/peer_actor.rs

```diff
@@ -0,0 +1,339 @@
+#![allow(unused_imports)]
+#![allow(dead_code)]
```

Comment:
I would be against blanket disabling of warnings, even for tests. What's the motivation here?

Reply:
good catch, removed